test/unittests/iotools/test_bids_utils.py

```diff
@@ -133,9 +133,9 @@ def create_clinical_data(tmp_path: Path, study_name: StudyName) -> Path:
             "OASIS": [np.nan, "ID", np.nan, "M/F", np.nan],
             "OASIS location": [
                 np.nan,
-                "oasis_cross-sectional.csv",
+                "oasis_cross-sectional-5708aa0a98d82080.xlsx",
```

Comment:
What is this number ? Is it constant and the same for all downloads and all users ?

Reply:
Found the answer in your linked issue, sorry for the noise.

Reply:
It's a fixed number/hash determined by OASIS. It does not seem to change overtime nor depending on user. For ex it is the exactly the same as when we first noticed the change approx 6 months ago.